code/index.html

```diff
@@ -1,22 +1,187 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&family=Noto+Serif+JP:wght@500;600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+    <link rel="stylesheet" href="style.css">
+
+    <title>POSITIVE NEWS</title>
+</head>
+
+<body>
+    <!-- HEADER -->
+    <header class="header-parent">
+        <div class= "logo-title">
+              <img src="Heart-speechbubbble.svg" alt="Image of positive news logo">
+              <h1>POSITIVE NEWS</h1>
+        </div>
+
+            <!-- Container for the hamburger -->
+            <div class="hamburger-menu">
+                  <div></div>
+                  <div></div>
+                  <div></div>
+            </div>
+
+            <nav>
+              <a href="#">HOME</a>
+              <a href="#">SECTIONS</a>
+              <a href="#">ABOUT</a>
+              <a href="#">CONTACT</a>
+            </nav>
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
+   <!-- MAIN -->
+    <main>
+        <!-- MAIN ARTICLE -->
+        <section class="main-article-parent">
+            <div class="main-article-child">
+                <img src="./Sea-and-Windfarm.jpg" alt="Image of wind farm at sea.">
+                <div class="text-area">
+                    <div class="container">
```

Comment:
Same thing here, perhaps a second container could be avoided by adding the styling to the text-area container instead 🙌